app/models.py/User.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.utils import timezone
+
+
+class User(AbstractUser):
```

Comment:
we need to make User has relation with AbstractUser or default User model  one2one
https://docs.djangoproject.com/en/5.0/topics/auth/customizing/#reusable-apps-and-auth-user-model